### PR TITLE
Added priority to offer limits

### PR DESCRIPTION
**Pull request by robertgresogitl**
Added priority to global, globalPerWindow, perProfile, perProfilePerWindow

### CODE REVIEW

File: Example/LMGRemoteData.xcworkspace/xcshareddata/WorkspaceSettings.xcsettings [hunk 1]
```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (cayleyh) [inline Example/LMGRemoteData.xcworkspace/xcshareddata/WorkspaceSettings.xcsettings]: Is this supposed to be committed?

File: LMGRemoteData/Classes/GraphQL/auth-schema.json [hunk 2]
```diff
@@ -314,6 +314,45 @@
                   "ofType": null
                 }
               }
+            },
```
Comment (cayleyh) [inline LMGRemoteData/Classes/GraphQL/auth-schema.json]: Why did auth-schema change?

File: Example/LMGRemoteData.xcworkspace/xcshareddata/WorkspaceSettings.xcsettings (hunk 1 above)
Comment (robertgresogitl) [reply Example/LMGRemoteData.xcworkspace/xcshareddata/WorkspaceSettings.xcsettings]: I dont think so, I fixed it.

File: LMGRemoteData/Classes/GraphQL/auth-schema.json (hunk 2 above)
Comment (robertgresogitl) [reply LMGRemoteData/Classes/GraphQL/auth-schema.json]: My mistake, fixed it as well.

File: Example/LMGRemoteData.xcworkspace/xcshareddata/WorkspaceSettings.xcsettings (hunk 1 above)
Comment (cayleyh) [reply Example/LMGRemoteData.xcworkspace/xcshareddata/WorkspaceSettings.xcsettings]: Can you add the `xcshareddata/` folder to the `.gitignore` so these don't get accidentally added again please?

**Pull request merged**